website/payments/api/v2/admin/serializers/payable_detail.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework.fields import CharField, empty, ListField, DecimalField
+from rest_framework.serializers import Serializer
+
+from members.api.v2.serializers.member import MemberSerializer
+from payments.api.v2.admin.serializers.payment import PaymentSerializer
+from payments.models import Payment
+from payments.payables import Payable
+
+
+class PayableSerializer(Serializer):
+    """Serializer to show payable information."""
+
+    def __init__(self, instance=None, data=empty, **kwargs):
+        super().__init__(instance, data, **kwargs)
+        self.allowed_payment_types = [Payment.CASH, Payment.CARD, Payment.WIRE]
+        if instance:
+            self.amount = instance.payment_amount
+            self.payer = instance.payment_payer
+            self.topic = instance.payment_topic
+            self.notes = instance.payment_notes
+            self.payment = instance.payment
+
+            if instance.tpay_allowed:
+                self.allowed_payment_types.append(Payment.TPAY)
+
+    allowed_payment_types = ListField(child=CharField())
+    amount = DecimalField(decimal_places=2, max_digits=1000)
```

Comment:
I think we're using `max_digits=6` everywhere (perhaps we should make a constant for this in the the settings)

Reply:
```suggestion
    amount = DecimalField(decimal_places=2, max_digits=6)
```
no clue what it actually best would be, 5/6/7 🤷